playfab-docs/features/multiplayer/servers/quickstart-for-multiplayer-servers-api-powershell.md

```diff
@@ -91,7 +100,7 @@ Once we see some standing by servers, let's request one for gameplay...
 $Regions = new-object 'System.Collections.Generic.List[PlayFab.MultiplayerModels.AzureRegion]'
 $Regions.Add("EastUS");
 
-$svr = New-PFMultiplayerServer -BuildName "MyBuild" -SessionId "00000000-0000-0000-0000-000000000001" -SessionCookie "test cookie" -PreferredRegions $regions
+$svr = New-PFMultiplayerServer -BuildName "PSTest_built" -SessionId "00000000-0000-0000-0000-000000000001" -SessionCookie "test cookie" -PreferredRegions $regions
```

Comment:
it shouldn't be too hard to put a in a proper GUID here so that it doesn't encourage weird behavior.

Reply:
Agreed.